test/images/agnhost/webhook/pods.go

```diff
@@ -30,6 +30,9 @@ const (
 	podsInitContainerPatch string = `[
 		 {"op":"add","path":"/spec/initContainers","value":[{"image":"webhook-added-image","name":"webhook-added-init-container","resources":{}}]}
 	]`
+	podsSidecarPatch string = `[
+		{"op":"add", "path":"/spec/containers/-","value":{"image":"k8s.gcr.io/pause:3.1","name":"webhook-added-sidecar","resources":{}}}
```

Comment:
if you're actually wanting this to run, I'm not sure whether hard-coding `k8s.gcr.io` images matches expectations for sig-testing and the ability to run e2e tests offline or from alternate image registries. also, wouldn't running a pause container cause the pod to wait for the full grace period on termination?

Reply:
I'm confused... So e2e tests should use not existing images? I mean "webhook-added-image" is a random name and not existing image, right? Or is this alias to something?

Reply:
I want to use this sidecar injector in vertical pod autoscaling E2E tests. The assumption is that we have real pods pods running. We are using `k8s.gcr.io/ubuntu-slim:0.1` so I can use the same image here (if it helps).

Reply:
the registry used for e2e images is configurable, in e2e.test there's a registry of images used with a mapping to what actual image to reference

Reply:
i'm not overly familiar with this but I think @alejandrox1 might be?
we generally want to make it possible to use images from an airgapped registry to e.g. conformance test

Reply:
I think you want to use `GetPauseImageName()` instead of hardcoding

Reply:
ref: https://github.com/kubernetes/kubernetes/blob/81acde7adc3a94dab76aea9e4df4f38c6013c2e1/test/utils/image/manifest.go#L278

Reply:
ok right, the registry used by e2e.test isn't available inside of agnhost, in which case I would pass in the image name to be used to `agnhost webhook` as a command line parameter

Reply:
`sidecar-image` parameter added.